### PR TITLE
Update to 1.2.0

### PR DESCRIPTION
**Pull request by Kekun**

### CODE REVIEW

File: com.adrienplazas.Metronome.json [hunk 1]
```diff
@@ -34,63 +34,14 @@
         "/bin/sassc"
```
Comment (Exalm) [inline com.adrienplazas.Metronome.json:34]: This looks like it caxn go away.

**Comment by hfiguiere** [comment]
seems that it is missing some crates.

**Comment by A6GibKm** [comment]
Hello, any update to this? I think it is the only app listed in Apps for GNOME using 40.

**Comment by Kekun** [comment]
bot build

**Comment by hfiguiere** [comment]
on PR, it starts automatically. And if it says "Queued" then the bot is on it. There was some lag today.

And for the politiness, the bot wants a `,`.

**Comment by hfiguiere** [comment]
build fail because it want to checkout something at build time.

**Comment by A6GibKm** [comment]
I sent [this MR](https://gitlab.gnome.org/World/metronome/-/merge_requests/28) to metronome, ran meson-dist on Builder and I was able to produce a tarball which builds just fine with flatpak-builder.

I also suggest the following changes

```diff
modified   com.adrienplazas.Metronome.json
@@ -9,15 +9,13 @@
     "command": "metronome",
     "finish-args" : [
         "--device=dri",
+        "--share=ipc",
         "--socket=fallback-x11",
         "--socket=pulseaudio",
         "--socket=wayland"
     ],
     "build-options" : {
-        "append-path" : "/usr/lib/sdk/rust-stable/bin",
-        "env" : {
-            "CARGO_HOME" : "/run/build/metronome/cargo"
-        }
+        "append-path" : "/usr/lib/sdk/rust-stable/bin"
     },
     "cleanup" : [
         "/include",
```

as apps running on x11 see benefits somehow from `ipc` and the `CARGO_HOME` variable is not needed, at least once my MR is in, I cannot say if it was ever needed though.

@bilelmoussaoui I think the problem was *specifically* fixed after setting both gtk-rs and libadwaita-rs to their last stable release in crates.io instead of `git`, do you have an idea of what might had caused the issue in the first time? We have certainly shipped stuff before using git.

**Comment by bilelmoussaoui** [comment]
> @bilelmoussaoui I think the problem was specifically fixed after setting both gtk-rs and libadwaita-rs to their last stable release in crates.io instead of git, do you have an idea of what might had caused the issue in the place? We have certainly shipped stuff before using git.

I think that issue was fixed already on gtk-rs side.

**Comment by hfiguiere** [comment]
Build will fail because the package to a crate replacement to grab a git repository... in offline mode.

**Comment by hfiguiere** [comment]
if only there was a 1.2.1 version with vendored crates...

**Comment by nud** [comment]
I guess you can update to runtime 44 now :smile:

**Comment by FineFindus** [comment]
[Superseded by version 1.2.1](https://github.com/flathub/com.adrienplazas.Metronome/pull/6)

**Pull request closed without merging**